pennylane/resource/measurement.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed for estimating the number of measurements and the error
+for computing expectation values.
+"""
+from pennylane import numpy as np
+
+
+def estimate_samples(coeffs, variances=None, error=0.0016):
+    r"""Estimate the number of measurements required to compute an expectation value with a target
+    error.
+
+    Args:
+        coeffs (list[tensor_like]): list of coefficient groups
+        variances (list[float]): variances of the Pauli word groups
+        error (float): target error in computing the expectation value
+
+    Returns:
+        int: the number of measurements
+
+    **Example**
+
+    >>> coeffs = [np.array([-0.32707061, 0.7896887]), np.array([0.18121046])]
```

Comment:
You could also add a "see also" section and link the other function `estimate_error` as they are closely related. (do something similar in the doc-string of the other function)